dist/global.css

```diff
@@ -0,0 +1,18 @@
+html {
```

Comment:
see comment below - can we make a css file like this shared instead of plugin-specific?